### PR TITLE
Using 'nameof' operator instead of magic string

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: test/Microsoft.AspNet.SignalR.SqlServer.Tests/ObservableSqlOperationFacts.cs [hunk 1]
```diff
@@ -119,7 +120,7 @@ public void CallsOnErrorOnException()
             var logger = new Mock<ILogger>();
             dbBehavior.Setup(db => db.UpdateLoopRetryDelays).Returns(_defaultRetryDelays);
             dbBehavior.Setup(db => db.StartSqlDependencyListener()).Returns(false);
-            dbProviderFactory.MockDataReader.Setup(r => r.Read()).Throws(new ApplicationException("test"));
+            dbProviderFactory.MockDataReader.Setup(r => r.Read()).Throws(new ApplicationException(nameof(test)));
```
Comment (moozzyk) [inline test/Microsoft.AspNet.SignalR.SqlServer.Tests/ObservableSqlOperationFacts.cs]: I don't see a `test` type/variable in scope - does it even compile?

File: src/Microsoft.AspNet.SignalR.SqlServer/SqlScaleoutOptions.cs [hunk 2]
```diff
@@ -27,13 +27,9 @@ public string ConnectionString
             {
                 return _connectionString;
             }
+            [param: NotNull]
             set
             {
-                if (String.IsNullOrEmpty(value))
```
Comment (Eilon) [inline src/Microsoft.AspNet.SignalR.SqlServer/SqlScaleoutOptions.cs:32]: Empty string != null...

File: test/Microsoft.AspNet.SignalR.SqlServer.Tests/ObservableSqlOperationFacts.cs (hunk 1 above)
Comment (Eilon) [inline test/Microsoft.AspNet.SignalR.SqlServer.Tests/ObservableSqlOperationFacts.cs]: Yeah "test" here is just an arbitrary string...

**Comment by hishamco** [comment]
@Eilon I update the PR to use `nameof` and left the `NotNull` while I have seen @pranavkm applying a script to throw `ArgumentNullException` instead

**Comment by pranavkm** [comment]
We're no longer building this repo.

**Comment by hishamco** [comment]
This was very old PR 😄

**Pull request closed without merging**